packages/terra-list/src/Section.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ListSectionHeader from './ListSectionHeader';
+import ListItem from './ListItem';
+
+const propTypes = {
+  /**
+   * The header content to be placed for display in a section.
+   */
+  headerContent: PropTypes.string,
+  /**
+   * The list of elements to be placed for display in a section.
+   */
+  listItems: PropTypes.array,
+};
+
+const defaultProps = {
+  headerContent: '',
+  listItems: [],
+};
+
+class Section extends React.Component {
+  constructor(props) {
+    super(props);
+    this.generateHeader = this.generateHeader.bind(this);
+  }
+
+  generateHeader() {
+    return <ListItem content={<ListSectionHeader id={this.props.headerContent} title={this.props.headerContent} onClick={this.handleClick} />} key={this.props.headerContent} />;
```

Comment:
Why does the id use the headerContent value? HTML ids can’t contain any space characters and this could result in id with space characters.